### PR TITLE
Spec requests

### PR DESCRIPTION
**Pull request by adeola003**
In this milestone, I created spec request for users and posts controllers and made sure:

- The response status was correct.
- A correct template was rendered.
- The response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
```
Comment (t-yanick) [inline spec/requests/posts_spec.rb]: - [ ] `index` and `show` actions for `User` and `Post` controllers should be tested like below:

```
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
```
Comment (t-yanick) [inline spec/requests/users_spec.rb]: - [ ] `index` and `show` actions for `User` and `Post` controllers should be tested like below:

```
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @adeola003 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Uses RSpec gem for testing
:heavy_check_mark: All test cases passing
:heavy_check_mark: Use of correct Gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @adeola003 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Implemented change requests to use the correct format for the tests and ensure all tests pass.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**